eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/recovery/MonotonicAccessionRecoveryAgentCategoryRSService.java

```diff
@@ -0,0 +1,28 @@
+package uk.ac.ebi.eva.accession.clustering.batch.recovery;
+
+import org.springframework.batch.core.JobExecution;
+import uk.ac.ebi.ampt2d.commons.accession.generators.monotonic.MonotonicAccessionRecoveryAgent;
+
+import java.time.LocalDateTime;
+
+public class MonotonicAccessionRecoveryAgentCategoryRSService {
```

Comment:
Just because we promised we'd argue about names... what about something shorter like `RSAccessionRecoveryService`? And analogous naming throughout the other classes.

Reply:
updated